backend/impl/src/test/resources/version.txt

```diff
@@ -1 +1 @@
-git-machete version 3.14.2
+git-machete version 3.15.1
```

Comment:
Not yet released (actually, I used the state from https://github.com/VirtusLab/git-machete/pull/813), but I don't think the outputs will have changed by the time of the actual release (soonish)

Reply:
Update: released already